packages/orchestration/README.md

```diff
@@ -245,24 +245,35 @@ Use the orchestration client with filtering to restrict content that is passed t
 
 This feature allows filtering both the [input](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/input-filtering) and [output](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/output-filtering) of a model based on content safety criteria.
 
-```ts
-import {
-  OrchestrationClient,
-  buildAzureContentFilter
-} from '@sap-ai-sdk/orchestration';
+#### Azure Content Filter
+
+Use `buildAzureContentSafetyFilter()` function to build an Azure content filter.
+The Azure content filter supports four categories: `Hate`, `Violence`, `Sexual`, and `SelfHarm`.
+Each category can be configured with severity levels of 0, 2, 4, or 6.
+
+Here is a complete example of using an Azure content filter for both input and output:
```

Comment:
[q] Is this line even needed? I feel like we're way to verbose sometimes when things are self-explanatory. We can just say (in the line above):

Use `buildAzureContentSafetyFilter()` function to build an Azure content filter for both input and output.